week-3-practise/level-2/2-map.js

```diff
@@ -14,7 +14,14 @@ var attendanceCounts = [
   ["Nina", 10]
 ];
 
-var eligibleStudentNames; // TODO: Complete this line.
+function studentWhoAttendedAtLeast8Classes(attendance){
+  return attendance[1] >= 8;
+}
+
+function studentNameWhoAttendedAtLeast8Classes(studentName){
+  return studentName[0];
```

Comment:
I would just call this function `studentName` and call the parameter `student`. All it does is get the name from the student data. It doesn't have anything to do with the number of classes attended.

Reply:
sure I will commit the change